docs/core/containers/overview.md

```diff
@@ -0,0 +1,207 @@
+---
+title: .NET SDK container creation overview
+description: Learn about the .NET SDK container creation feature, including telemetry, publishing considerations, and build properties.
+ms.date: 01/07/2025
+ms.topic: overview
+---
+
+# .NET SDK container creation overview
+
+While it's possible to [containerize .NET apps with a _Dockerfile_](../docker/build-container.md), there are compelling reasons for [containerizing apps directly with the .NET SDK](sdk-publish.md). This article provides an overview of the .NET SDK container creation feature, with details related to telemetry, publishing considerations, build properties, and authentication to container registries.
+
+## Telemetry
```

Comment:
nit: I'm not sure that telemetry disclosures should be the first thing users hit - can we move this below the other items?